app/src/main/java/org/oppia/app/drawer/NavigationDrawerFragment.kt

```diff
@@ -0,0 +1,91 @@
+package org.oppia.app.drawer
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.MenuItem
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.ActionBarDrawerToggle
+import androidx.appcompat.widget.Toolbar
+import androidx.drawerlayout.widget.DrawerLayout
+import androidx.fragment.app.Fragment
+import com.google.android.material.navigation.NavigationView
+import org.oppia.app.R
+import org.oppia.app.help.HelpActivity
+import org.oppia.app.home.HomeActivity
+
+/** [NavigationDrawerFragment] to show navigation drawer */
```

Comment:
Add punctuations to the comments like below
/** [NavigationDrawerFragment] to show navigation drawer. */

Reply:
okay